### DISCUSSION TITLE
Is there a way to access the id token from the login?

### DISCUSSION THREAD

**Discussion by jlangree** [Budibase/budibase#2727] [Ideas]
We are trying to hit our API, which has authorization rules set up based on the id token passed in the Auth header. If we integrate with Auth0 for the login process, would there be any way to access the token generated so we can use it for our REST API connection?

**Comment by mjashanks** [comment]
That is not possible at the moment, but I'm going to link to this discussion: https://github.com/Budibase/budibase/discussions/2726, as your suggestion sounds like a good idea to me.
